packages/xs-vat-worker/compartmap.json

```diff
@@ -0,0 +1,127 @@
+          "module": "./src/promiseKit.js"
+        },
+        "@agoric/nat": {
+          "compartment": "node_modules/@agoric/nat/",
+          "module": "./dist/nat.esm.js"
+        }
+      }
+    },
+    "node_modules/@agoric/nat/": {
+      "label": "@agoric/nat@2.0.1",
+      "location": "node_modules/@agoric/nat/",
+      "contents": [
+        "./dist/nat.esm.js"
+      ],
+      "modules": {}
+    },
+    "node_modules/@agoric/promise-kit/": {
+      "label": "@agoric/promise-kit@0.1.3",
+      "location": "node_modules/@agoric/promise-kit/",
+      "contents": [
+        "./src/promiseKit.js"
+      ],
+      "modules": {}
+    },
+    "node_modules/@agoric/assert/": {
+      "label": "@agoric/assert@0.0.8",
+      "location": "node_modules/@agoric/assert/",
+      "contents": [
+        "./src/assert.js",
+        "./src/types.js"
+      ],
+      "modules": {}
+    }
+  },
+  "modules": {
+    "node_modules/0_MKDIR": "$(ROOT)/node_modules/0_MKDIR",
+    "node_modules/@agoric/0_MKDIR": "$(ROOT)/node_modules/@agoric/0_MKDIR",
+    "node_modules/@agoric/assert/0_MKDIR": "$(ROOT)/node_modules/@agoric/assert/0_MKDIR",
+    "node_modules/@agoric/assert/src/assert": "$(ROOT)/node_modules/@agoric/assert/src/assert",
+    "node_modules/@agoric/assert/src/types": "$(ROOT)/node_modules/@agoric/assert/src/types",
+    "node_modules/@agoric/import-bundle/0_MKDIR": "$(ROOT)/node_modules/@agoric/import-bundle/0_MKDIR",
+    "node_modules/@agoric/import-bundle/src/compartment-wrapper": "$(ROOT)/node_modules/@agoric/import-bundle/src/compartment-wrapper",
+    "node_modules/@agoric/import-bundle/src/index": "$(ROOT)/node_modules/@agoric/import-bundle/src/index",
+    "node_modules/@agoric/marshal/marshal": "$(ROOT)/node_modules/@agoric/marshal/marshal",
+    "node_modules/@agoric/nat/0_MKDIR": "$(ROOT)/node_modules/@agoric/nat/0_MKDIR",
+    "node_modules/@agoric/nat/dist/nat.esm": "$(ROOT)/node_modules/@agoric/nat/dist/nat.esm",
+    "node_modules/@agoric/promise-kit/0_MKDIR": "$(ROOT)/node_modules/@agoric/promise-kit/0_MKDIR",
+    "node_modules/@agoric/promise-kit/src/promiseKit": "$(ROOT)/node_modules/@agoric/promise-kit/src/promiseKit",
+    "node_modules/@agoric/swingset-vat/0_MKDIR": "$(ROOT)/node_modules/@agoric/swingset-vat/0_MKDIR",
+    "node_modules/@agoric/swingset-vat/src/0_MKDIR": "$(ROOT)/node_modules/@agoric/swingset-vat/src/0_MKDIR",
+    "node_modules/@agoric/swingset-vat/src/capdata": "$(ROOT)/node_modules/@agoric/swingset-vat/src/capdata",
+    "node_modules/@agoric/swingset-vat/src/kernel/liveSlots": "$(ROOT)/node_modules/@agoric/swingset-vat/src/kernel/liveSlots",
+    "node_modules/@agoric/swingset-vat/src/parseVatSlots": "$(ROOT)/node_modules/@agoric/swingset-vat/src/parseVatSlots",
+    "packages/0_MKDIR": "$(ROOT)/packages/0_MKDIR",
+    "packages/xs-vat-worker/0_MKDIR": "$(ROOT)/packages/xs-vat-worker/0_MKDIR",
+    "packages/xs-vat-worker/src/vatWorker": "$(ROOT)/packages/xs-vat-worker/src/vatWorker"
+  }
+}
```

Comment:
any chance we could add a newline here?

Reply:
oh, wait, is this generated during the build? then we shouldn't check it in, and it should be added to `.gitignore` (and I'm less concerned about adding a newline)

Reply:
I'm hesitant to rely on the tool that builds it.

Reply:
Oh... but changes _outside_ this `xs-vat-worker` package can necessitate a rebuild of `compartmap.json` using `findmods.js`. This argues for taking `findmods.js` more seriously as a solution to #57. I'd like to see that it shares a typescript type for the compartmap with endo.

Reply:
If `compartmap.json` is only rebuilt manually, and most of the time we use the pre-built copy, then yeah we can check it in. But you're right, we need to think through when exactly it must be rebuilt (if/when swingset files get moved around, or liveslots.js acquires a new `import`?), and figure out some rules for rebuilding it. It'd be ok if those rules are "if you make changes to liveslots.js, re-run this command in `xs-vat-worker` and commit the resulting `compartmap.json`", but if the tool is stable enough, then it'd be even better if `xs-vat-worker` did this at build time. 

OTOH, like our top-level `yarn.lock` file, there are changes someone might make to the source tree that will invalidate this file, and it's too expensive to automatically rebuild it on every single `yarn test` or other operation. OT3H, our general rule is "if something looks weird, your first reaction should be to re-run `yarn (install)` and `yarn build`", so making `yarn build` regenerate the compartmap would take care of that well enough.

Reply:
It ain't broke as it currently stands, for the purposes of this PR.

I'm inclined to let #57 stand for the work of thinking harder about it.